templates/procuration_manager/_proposals_list.html.twig

```diff
@@ -27,18 +27,22 @@
             </span>
         </td>
         <td class="datagrid__table__col--left">
-            {{ include('procuration_manager/_matching_election_rounds.html.twig', {
-                proxy_rounds: proxy.availableRounds,
-                request: proxy.foundRequest,
-            }, with_context=false) }}
+            {{ proxy.getAvailableRoundsAsString|raw }}
```

Comment:
you can keep `implode("\n", ...)` on the methode and use here `|nl2br` twig filter